### PR TITLE
[VIRTS-1890] Plugin-specific documentation

### PR DESCRIPTION
**Pull request by b17zr**
Adding plugin-specific documentation to skeleton

### CODE REVIEW

File: plugin-init.py [hunk 1]
```diff
@@ -44,14 +44,17 @@ def rewrite_files(files, name, description):
...
         update_file_contents(file, search_text='description = \'description\'',
                              replace_text=f"description = '{description}'")
+        update_file_contents(file, search_text='Plugin documentation', replace_text=description)
 
 
 def rename_files(files, name):
     for file in files:
-        os.rename(file, file.replace('skeleton', name.lower()))
+        if 'Skeleton' in file:
+            os.rename(file, file.replace('Skeleton', name.capitalize()))
```
Comment (bworrell) [inline plugin-init.py:55]: Why this over having the markdown filename be `skeleton.md`?

Comment (b17zr) [reply plugin-init.py:55]: Clearly figured writing 4 lines of Python was easier than one `mv`

File: docs/skeleton.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+# Skeleton
+
+Plugin documentation
```
Comment (bworrell) [inline docs/skeleton.md:4]: I would add some notes in here about how fieldmanual will look for this directory and pull in any files. Or add it to the README.

**Review by bworrell — CHANGES_REQUESTED** [review]
Let's mention somewhere that the `docs` folder contents will be loaded into fieldmanual if it's enabled.

File: docs/skeleton.md (hunk 2 above)
Comment (b17zr) [reply docs/skeleton.md:4]: After merging the big fieldmanual changes, would it be appropriate to add this under the Plugins section there, instead?

**Review by bworrell — APPROVED** [review]
LGTM!

**Pull request merged**